internal/storage/storage_handle.go

```diff
@@ -35,11 +35,11 @@ type storageClient struct {
 	client *storage.Client
 }
 
-type storageClientConfig struct {
+type StorageClientConfig struct {
 	disableHTTP2        bool
 	maxConnsPerHost     int
 	maxIdleConnsPerHost int
-	tokenSrc            oauth2.TokenSource
```

Comment:
why is this made public?

Reply:
because while calling NewStorageHandle we need to set particular token in storageclientConfig, as before it was private was not able to set it up.

Reply:
others fields also need to be set right from outside. Why are they not public

Reply:
because we need to set on only tokensrc only, that's why I made chnages in that variable only

Reply:
please update the comment and resolve it.

Reply:
made all fields public. 
fixed